contracts/src/v0.8/dev/CanaryUpkeep.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity 0.8.6;
 
 import "../interfaces/KeeperCompatibleInterface.sol";
-import "../interfaces/KeeperRegistryInterface.sol";
+import "../interfaces/KeeperRegistryInterface1_2.sol";
```

Comment:
Should we rename CanaryUpkeep to CanaryUpkeep1_2 ?

Reply:
Done